.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+f2f767d08775e7834000a84b7bdf1dad62c8acbb
```

Comment:
This commit does not seem to be in the history of this PR, should it be the "black formatting" one? `4b4053c24f4dd144eb755c190c1b6bafff16d606`